resources/views/content/static-data.blade.php

```diff
@@ -24,32 +24,62 @@
 $lastRegisteredUserAt = $staticData['LastRegisteredUserAt'];
 $totalPointsEarned = $staticData['TotalPointsEarned'];
 
-$niceRegisteredAt = date("d M\nH:i", strtotime($lastRegisteredUserAt));
+$lastRegisteredUserAtDate = new DateTime($lastRegisteredUserAt);
+$now = new DateTime();
+$interval = $lastRegisteredUserAtDate->diff($now);
+
+$timeUnits = [
+    "y" => "year",
+    "m" => "month",
+    "d" => "day",
+    "h" => "hour",
+    "i" => "minute",
+    "s" => "second",
+];
+foreach ($timeUnits as $format => $unit) {
+    if ($interval->$format >= 1) {
+        $unit = $interval->$format > 1 ? $unit . "s" : $unit;
+        // "%$format $unit ago" --> "5 minutes ago"
+        $lastRegisteredUserTimeAgo = $interval->format("%$format $unit ago");
+        break;
+    }
+}
```

Comment:
Carbon has a way to generate these nicely formatted elapsed times:
https://github.com/RetroAchievements/RAWeb/pull/1321/files#diff-81cb00a9d43fff3bd73a0f74426506b9ddeec2eab6d9d72d210d26a9381fa39eR13

Reply:
That saves a lot of code, thanks. Fixed.

I had to add a truthiness check in the blade to keep a test case from failing.